src/wallet/gtest/test_wallet.cpp

```diff
@@ -1238,8 +1246,10 @@ TEST(WalletTests, CachedWitnessesEmptyChain) {
 }
 
 TEST(WalletTests, CachedWitnessesChainTip) {
+    SelectParams(CBaseChainParams::REGTEST);
```

Comment:
```suggestion
    (void) RegtestActivateSapling();
```

Reply:
This test appears at first sight to be testing Sapling, but isn't. It should probably be extended (not in this PR).

Reply:
Yup. I went through each of the tests and explicitly only changed those tests where they would actually fail if Sapling were not activated.